proposals/vitess.adoc

```diff
@@ -0,0 +1,109 @@
+== Vitess
+
+*Name of project*: Vitess
+
+*Description*:
+
+Vitess is a database clustering system for horizontal scaling of MySQL. Using the terminology from the link:http://db.cs.cmu.edu/papers/2016/pavlo-newsql-sigmodrec2016.pdf[Pavlo and Aslett NewSQL survey article], Vitess is “sharding middleware”.  By encapsulating shard-routing logic, Vitess allows application code and database queries to remain agnostic to the distribution of data onto multiple shards. You can split and merge shards as your needs change, with an atomic cutover step that is performed in seconds. Vitess has been serving all YouTube database traffic since 2011, and has grown to encompass tens of thousands of MySQL nodes. It has also gained increasing adoption in the community with about fifteen companies currently in the pipeline, some of whom have already gone into production. For more details, see the link:http://vitess.io/overview/[Vitess overview].
```

Comment:
There was a request to use the term "orchestration" here. I propose inserting the following sentence after the first:

"Vitess orchestrates management of MySQL instances and intermediates requests to the cluster."

and the following one (borrowed from the Vitess overview), just before the sentence about serving YouTube traffic:

"Vitess also supports and automatically handles various scenarios, including master failover and data backups."

since that functionality is key to operating in a cloud-native environment -- Vitess is about more than just scaling and sharding.

Reply:
Done.